pages/mesosphere/dcos/1.13/release-notes/1.13.8/index.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: layout.pug
+navigationTitle: Release notes for 1.13.8
+title: Release notes for 1.13.8
+menuWeight: 0
+excerpt: Release notes for DC/OS 1.13.8, including Open Source attribution, and version policy.
+---
+DC/OS&trade; 1.13.8 was released on ?? March 2020.
+
+[button color="light" href="https://downloads.dcos.io/dcos/stable/1.13.8/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="purple" href="https://downloads.mesosphere.com/dcos-enterprise/stable/1.13.8/dcos_generate_config.ee.sh"]Download DC/OS Enterprise* [/button]
+
+Registered DC/OS Enterprise customers can access the DC/OS Enterprise configuration file from the [support website](https://support.mesosphere.com/s/downloads). For new customers, contact your sales representative or <a href="mailto:sales@mesosphere.io">sales@mesosphere.io</a> before attempting to download and install DC/OS Enterprise.
+
+# Release Summary
+DC/OS is a distributed operating system that enables you to manage resources, application deployment, data services, networking, and security in an on-premise, cloud, or hybrid cluster environment.
+
+<!--- Baker took the decision on 3/10/2020 to order the entries under each subject heading in descending COPS ticket order, for ease of checking against Margaret Atamar's release deashboard. --->
+
+# DC/OS 
+
+## Components
+
+DC/OS 1.13.8 includes the following component versions:
+
+- Apache&reg; Mesos&reg; 1.8.2-dev
+
+### DC/OS Fixed and Improved Issues
+
+- Fixed an issue where in some rare circumstances, after upgrading a cluster from DC/OS 1.11 to DC/OS 1.13 users were no longer able to launch tasks that use the UCR containerizer. (D2IQ-64507, COPS-5868)
+
+- DC/OS no longer increases the rate limit for `journald` logging. Scale testing demonstrated that raising the limit can overload `journald`, causing stress for other components. The default of 10000 messages per 30 seconds appears to distinguish well between busy components and excessively verbose components. (D2IQ-53763, COPS-5830)
+
+- Fixed an issue where image pull in UCR was not working for nvcr.io (missing 'service'/'scope' parameters). (D2IQ-63303, COPS-5804)
+
+- Fixed situation where application on MoM always waiting because the master does not allocate valid offers to framework. (D2IQ-62519, COPS-5725)
```

Comment:
s/always/was always/ ?

Reply:
Even with Vinod's excellent suggestion this is pretty hard to understand...

How about this instead:

"Fixed an issue where in some rare circumstances, the executor resources used by tasks on the agent were being counted against quota when they should not have been".  (D2IQ-62519, COPS-5725)    <--- What do you think @vinodkone ?

Reply:
How about this:

"Fixed an issue where after a DCOS upgrade, executor resources used by tasks were incorrectly counted against quota. (D2IQ-62519, COPS-5725)"

Reply:
Final: Fixed an issue where after a DC/OS upgrade, the executor resources used by tasks on the agent were being incorrectly counted against quota. (D2IQ-62519, COPS-5725)